### ISSUE TITLE
Taskbar won't update

### ISSUE THREAD

**Issue by OneGunBullet** [torchgm/RoundedTB#118]
![image](https://user-images.githubusercontent.com/70110430/143625023-cee3a407-921e-445a-a1f3-fb7b5bb84f98.png)

**Comment by dongle-the-gadget** [comment]
Huh. Is "Fill taskbar on maximized" on?

**Comment by OneGunBullet** [comment]
> Huh. Is "Fill taskbar on maximized" on?

It's enabled in the picture yes

**Comment by dongle-the-gadget** [comment]
> > Huh. Is "Fill taskbar on maximized" on?
> 
> It's enabled in the picture

Yeah that's a bug with 3.1 which I heard has been fixed, just not pushed.

**Comment by OneGunBullet** [comment]
> 

it happens when I have it off too though... and with any setting applied

**Comment by dongle-the-gadget** [comment]
> > 
> 
> it happens when I have it off too though... and with any setting applied

Oh, weird.

**Comment by torchgm** [comment]
Are you using any other taskbar modifications such as TranslucentTB or TaskbarX?

**Comment by OneGunBullet** [comment]
> 

No. I actually checked what would happen if I did, but the result was the same.
